readme.md

```diff
@@ -33,72 +33,104 @@ To read further about the file generation features for each year, please refer t
 
 For 2019: 
-1. Run `cd 2019/python` to navigate to the python directory. 
+1. Navigate to the 2019/python directory.
 2. Run `python3 generate_2019_error_files.py`
-3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+3. The error files for testing syntax, validity, and quality edit test files will be created in the following diretories:
 	- Syntax: `2019/edits_files/test_files/{Bank Name}/syntax`
 	- Validity: `2019/edits_files/test_files/{Bank Name}/validity`
 	- Quality: `2019/edits_files/test_files/{Bank Name}/quality`
 
 Note: the filepath and filename for test files can be adjusted in the [2019 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`.  
 
 For 2018: 
-1. Run `cd 2018/python` to navigate to the python directory. 
+1. Navigate to the 2018/python directory.
 2. Run `python3 generate_2018_error_files.py`
-3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+3. The error files for testing syntax, validity, and quality edit test files will be created in the following diretories:
 	- Syntax: `2018/edits_files/test_files/{Bank Name}/syntax`
 	- Validity: `2018/edits_files/test_files/{Bank Name}/validity`
 	- Quality: `2018/edits_files/test_files/{Bank Name}/quality`
 
 Note: the filepath and filename for test files can be adjusted in the [2018 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`. 
 
 ## Generating Large Files 
-Often, large files would need to be generated to support load testing. The repository allows users to creates a test file based on existing test file data to create a new file with a larger number of rows.
+Large files are used during load testing. Large test files are created using a clean test file base and the user-specified row count in each year's large file configuration:
+ [2019 Large File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml).
+ [2018 Large File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml).
+
+The values listed in each configuration file can be changed. The default configuration contains the following values:
+	`source_filepath: #Specify source filepath.  
```

Comment:
Check your formatting for newlines. Please preview when making these kinds of changes. GH uses a markdown variant that handles newlines differently.

Reply:
Sure - I am going to place these as bullets